admin_manual/maintenance/manual_upgrade.rst

```diff
@@ -2,83 +2,165 @@
 Manual ownCloud Upgrade
 =======================
 
-.. warning:: When upgrading from oC 9.0 to 9.1 with existing Calendars or Adressbooks
-   please have a look at the :doc:`../release_notes` of oC 9.0 for important info
-   about the needed migration steps during that upgrade.
-
-Always start by making a fresh backup and disabling all 3rd party apps.
-
-Put your server in maintenance mode. This prevents new logins, locks the 
-sessions of logged-in users, and displays a status screen so users know what is 
-happening. There are two ways to do this, and the preferred method is to use the 
-:doc:`occ command <../configuration/server/occ_command>`, which you must run as 
-your HTTP user. This example is for Ubuntu Linux::
-
- sudo -u www-data php occ maintenance:mode --on
- 
-The other way is by entering your ``config.php`` file and changing 
-``'maintenance' => false,`` to ``'maintenance' => true,``. 
-
-1. Back up your existing ownCloud Server database, data directory, and 
-   ``config.php`` file. (See :doc:`backup`.)
-2. Download and unpack the latest ownCloud Server release (Archive file) from 
-   `owncloud.org/install/`_ into an empty directory outside 
-   of your current installation.
+.. warning:: 
+   When upgrading from oC 9.0 to 9.1 with existing Calendars or Adressbooks, please have a look at the :doc:`../release_notes` of ownCloud 9.0 for important info about the needed migration steps during that upgrade.
```

Comment:
This is the same as another PR I just reviewed?
s/Adressbooks/Addressbooks

Reply:
I forgot to correct this one, in light of those changes.